renderdoc/driver/metal/metal_common.h

```diff
@@ -35,6 +35,7 @@
 enum class MetalChunk : uint32_t
 {
   MTLCreateSystemDefaultDevice = (uint32_t)SystemChunk::FirstDriverChunk,
+  MTLDevice_newCommandQueue,
```

Comment:
It doesn't matter yet so you don't have to change this in this PR, but note that it's good practice for later to always add new chunks at the end of the enum. Since these chunk values are serialised and used to look up which chunk to replay with, it is part of the backwards compatibility requirements to make sure they never change.